### PR TITLE
Update contacts titles

### PR DESCRIPTION
**Pull request by jmichlitsch**
### What Changed?
-Add branch main to travis yml
Add tests for contacts job titles
Add job title to model and serializers.
### What's Next?
- Add job title to csv files
### Known Issues?
None at the moment
### Test Coverage?
- 100%

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,4 +16,5 @@ deploy:
   app: qc-engine
   on:
     repo: Quick-Command/qc-engine
+    branch: main
```
Comment (klatour324) [inline .travis.yml:19]: Thanks for the fix here.  Hopefully this alleviates the issue we were seeing on Travis CI despite the fact that it was still deploying.

File: db/schema.rb [hunk 2]
```diff
@@ -21,6 +21,7 @@
     t.string "phone_number"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "job_title"
```
Comment (klatour324) [inline db/schema.rb:24]: Great catch with noticing this distinction on the wireframe. Let's make sure we also update the current csv file to include a column for job title as well.

**Review by klatour324 — APPROVED** [review]
Great fixes here Jeremiah! Thank you for noticing on the wireframes the job title they were specifying.

File: db/schema.rb (hunk 2 above)
Comment (amsmyth1) [reply db/schema.rb:24]: I will add the CSV with the next PR. Thanks for taking care of this J!

**Review by amsmyth1 — APPROVED** [review]
Great catch, Jeremiah! I will add the updated CSV in the next PR

**Pull request merged**